xapian-core/queryparser/cjk-tokenizer.h

```diff
@@ -36,6 +36,11 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wold-style-cast"
```

Comment:
These should be conditional on `#ifdef __GNUC__` to avoid "unknown pragma" warnings with other compilers.  Looks like support for these were [added in GCC 4.6](https://gcc.gnu.org/gcc-4.6/changes.html) so we don't need to worry about what happens with older GCC versions.

Reply:
Fixed in a858fcb